src/badguy/mole.cpp

```diff
@@ -75,14 +76,13 @@ Mole::collision_squished(GameObject& )
 void
 Mole::throw_rock()
 {
-  float angle;
-  float base_angle = (m_flip == NO_FLIP ? 90.0f : 270.0f);
-  angle = math::radians(gameRandom.randf(base_angle - 15.0f, base_angle + 15.0f));
-  float vx = cosf(angle) * THROW_VELOCITY;
-  float vy = -sinf(angle) * THROW_VELOCITY;
+  float angle = math::radians(135.f - (float(cycle_num) * 30.f));
```

Comment:
Use `static_cast` instead:

```suggestion
  float angle = math::radians(135.f - (static_cast<float>(cycle_num) * 30.f));
```

Reply:
Look! It's a giant commit button

Reply:
I want to be able to squash unrelated commit suggestions.